FitTracker/app/src/main/java/com/pps/globant/fittracker/mvp/model/DataBase/User.java

```diff
@@ -0,0 +1,31 @@
+package com.pps.globant.fittracker.mvp.model.DataBase;
+
+
+import android.arch.persistence.room.ColumnInfo;
+import android.arch.persistence.room.Entity;
+import android.arch.persistence.room.PrimaryKey;
+import android.support.annotation.NonNull;
+
+@Entity(tableName = User.TABLE_NAME)
+public class User {
+    private static final String NAME = "name";
+    static final String TABLE_NAME = "user";
+
+    public User(@NonNull String name) {
```

Comment:
I think you should remove all the parameters from the constructor, and directly remove the constructor and use the default constructor.

Reply:
@nacho-globant the constructor is necessary even with the annotations, this example :

@Entity(tableName = User.TABLE_NAME)
public class User {
    private static final String NAME = "name";
    static final String TABLE_NAME = "user";

    @NonNull
    @PrimaryKey
    @ColumnInfo(name = NAME)
    private String name;

    public String getName() {
        return name;
    }

    public void setName(String firstName) {
        this.name = firstName;
    }

}

does not compile

error: constructor User in class User cannot be applied to given types;
required: no arguments
found: String
reason: actual and formal argument lists differ in length

Reply:
ok. Thanks for the clarification